solr/core/src/test/org/apache/solr/handler/TestContainerPlugin.java

```diff
@@ -572,7 +572,7 @@ private V2Request postPlugin(Object payload) {
 
   public void waitForAllNodesToSync(String path, Map<String, Object> expected) throws Exception {
     for (JettySolrRunner jettySolrRunner : cluster.getJettySolrRunners()) {
-      String baseUrl = jettySolrRunner.getBaseUrl().toString().replace("/solr", "/api");
+      String baseUrl = jettySolrRunner.getBaseURLV2().toString();
```

Comment:
so many `.toString()` ;-).   I wonder...    Should we be working more with URL objects and less with String objects when we build our urls???